js/script.js

```diff
@@ -0,0 +1,79 @@
+var books = [];
+
+const form = document.querySelector('.book-form');
+const formId = document.querySelector('.book-form__input--id');
+const formTitle = document.querySelector('.book-form__input--title');
+const formEditionYear = document.querySelector('.book-form__input--year');
+const formAuthor = document.querySelector('.book-form__input--author');
+const formCost = document.querySelector('.book-form__input--cost');
+const formNumberOfCopies = document.querySelector('.book-form__input--copies');
+
+form.addEventListener('submit', (e) => {

```

Comment:
in case of multiple submit elements in a form this function will be triggered on all of the buttons. Can you create a  function for handling clicks on specific button ?